bin/i18n/i18n_script_utils.rb

```diff
@@ -148,6 +170,14 @@ def redact(source, dest, *plugins)
   end
 end
 
+def contains_malformed_link_or_image(translation)
```

Comment:
I'd love to see some comments explaining what these regexes are looking for and why we're looking for them

Reply:
Done!